### DISCUSSION TITLE
Next JS Page is blank

### DISCUSSION THREAD

**Discussion by emanuelavram** [vercel/community#4957] [Help] [answered]
### Page to Investigate

https://flairorder.vercel.app

### Steps to Reproduce

After my last publish my Next JS on Vercel returns an empty page.
It works locally and on my ubuntu server, only on vercel it doesn't.

I've already tried to remove index.html from public folder, "homepage" attribute from package.json

Can't find the issue.
Someone any ideas?

**Comment by amyegan** [comment, marked as answer]
I see some errors in the browser console that might be related:
<img width="747" alt="Error with permissions policy header" src="https://github.com/vercel/community/assets/8247105/12c66005-2338-4dbb-a0d8-f35c210e30bb">

**Reply by emanuelavram** [reply]
The first one (MUI) is a deprecation warning
The other warning may be vercel related
But i dont think they cause the error. Because i would expect an error not a warning

The build runs on my local machine (windows) and on ubuntu 22 LTS 
So i guess it has to do with some vercel config/...

**Reply by emanuelavram** [reply]
I deleted my version and reimported the project from github.
After that, i upgraded localizationProvider from mui/lab to x-date-pickers and the issue seems gone
Can't really imagine that this warning was the issue, but it works now.
